util/KoLmafia.importorder

```diff
@@ -1,4 +1,3 @@
-0=apple.dts.samplecode.osxadapter
 1=ca.bcit.geekkit
```

Comment:
Should this be zero-indexed, or does it not matter?

Reply:
Nah it just traverses in order I think